app/src/androidTest/java/net/skyscanner/backpack/BpkSnapshotTest.kt

```diff
@@ -61,34 +58,27 @@ open class BpkSnapshotTest(private val tags: List<Any> = emptyList()) : Screensh
       view = view,
       background = background,
       padding = padding,
-      wrapView = wrapView,
     )
     wrappedView.measure(measureSpec(width), measureSpec(height))
     compareScreenshot(
       view = wrappedView,
-      widthInPx = wrappedView.measuredWidth,
-      heightInPx = wrappedView.measuredHeight,
+      widthInPx = width?.let { dpToPx(it) } ?: wrappedView.measuredWidth,
+      heightInPx = height?.let { dpToPx(it) } ?: wrappedView.measuredHeight,
       name = screenshotName(tags),
     )
   }
 
   private fun measureSpec(size: Int?): Int {
-    val dimenSize = if (size == null) {
-      0
-    } else {
-      (size * testContext.resources.displayMetrics.density).toInt()
-    }
+    val dimenSize = size?.let { dpToPx(it) } ?: 0
     return makeMeasureSpec(dimenSize, if (size == null) View.MeasureSpec.UNSPECIFIED else View.MeasureSpec.EXACTLY)
   }
 
-  private fun wrapMeasuredViewWithBackground(view: View, background: Int, padding: Int, wrapView: Boolean): View {
-    if (!wrapView) {
-      return view
-    }
+  private fun wrapMeasuredViewWithBackground(view: View, background: Int, padding: Int): View {
     val result = FrameLayout(view.context)
     if (view.parent != null) {
       runOnUi {
         (view.parent as ViewGroup).removeView(view)
+        view.visibility = View.VISIBLE
```

Comment:
turns out that was the root cause of the snackbar tests not working with a wrapped view. the snackbar is set to invisible first for some reason. we never want to test invisble views anyways